.circleci/config.yml

```diff
@@ -1,25 +1,15 @@
-
 version: 2.1
-orbs:
-  node: circleci/node@4.7
-  cypress: cypress-io/cypress@2.1.0
+orbs: 
+  cypress: cypress-io/cypress@1
+executors: 
+  with-chrome:
+    docker:
+      - image: 'cypress/browsers:node14.16.0-chrome90-ff88'
 workflows:
-  build:
+  build: 
     jobs:
       - cypress/run:
-jobs:
-  build_and_test:
-    executor: node/default
-    steps:
-      - node/install-packages:
-          pkg-manager: npm
-      - run:
-          name: Run tests
-          command: npm run cy:run
-      - run :
-          name : Run build
-          command: npm run build
-      - persist_to_workspace:
-          root: ~/project
-          paths:
-            - .
+          executor: with-chrome
+          browser: chrome
+          start: npm start
+          wait-on: 'http://localhost:3000'
```

Comment:
Interesting! I tried the `wait-on: 'http://localhost:3000'` but didn't have the executor and browser. Nice find Trish.